user/deployment-v2/providers/elasticbeanstalk.md

```diff
@@ -22,4 +22,9 @@ deploy:
 ```
 {: data-file=".travis.yml"}
 
+## Controlling which files are included in the zip archive
```

Comment:
```suggestion
## Controlling which files are included in the .zip archive
```

Reply:
Personally I think "zip archive" is the right technical term, while ".zip" refers to the file extension?

Reply:
Personally I think "zip archive" is the right technical term, while ".zip" refers to the file extension?

Reply:
AWS seems to use `ZIP file`, so maybe let's follow that?